src/redux/reservations/deleteReservation.js

```diff
@@ -0,0 +1,75 @@
+const API_RESERVATIONS_INDEX_ENDPOINT = `${process.env.REACT_APP_API_HOST}/api/v1/reservations`;
+const REQUEST_STARTED = 'book-a-lawyer/reservations/REQUEST_STARTED';
+const REQUEST_FAILED = 'book-a-lawyer/reservations/REQUEST_FAILED';
+const REQUEST_COMPLETED = 'book-a-lawyer/reservations/REQUEST_COMPLETED';
+const initialState = {
+  reservations: [],
+};
```

Comment:
This part is exactly the same as the `reservation.js` reducer, the action types are the same, which may cause unexpected behaviors. A good idea is renaming the action types.
Also a good idea is having an initial status for the state.
```suggestion
const API_RESERVATIONS_DELETE_ENDPOINT = `${process.env.REACT_APP_API_HOST}/api/v1/reservations`;
const REQUEST_STARTED = 'book-a-lawyer/reservationDelete/REQUEST_STARTED';
const REQUEST_FAILED = 'book-a-lawyer/reservationDelete/REQUEST_FAILED';
const REQUEST_COMPLETED = 'book-a-lawyer/reservationDelete/REQUEST_COMPLETED';
const initialState = {
  reservations: [],
  status: 'idle',
};
```